scm-ui/ui-webapp/src/repos/containers/EditRepo.tsx

```diff
@@ -57,6 +57,7 @@ const EditRepo: FC<Props> = ({ repository }) => {
       <Subtitle subtitle={t("repositoryForm.subtitle")} />
       <ErrorNotification error={error} />
       <RepositoryForm repository={repository} loading={isLoading} modifyRepository={update} />
+      <ExtensionPoint name="repo-config.details" props={extensionProps} />
```

Comment:
Do we need a `renderAll={true}` here?

Reply:
Why not? There could be other plugins which want to provide configuration forms to the repo settings page?

Reply:
Ah, and please fix the storyshots.